functest/send_money_test.go

```diff
@@ -65,10 +66,10 @@ func TestTransferMoneyFromNotExist(t *testing.T) {
 	secondMember := createMember(t, "Member2")
 	oldSecondBalance := getBalanceNoErr(t, secondMember, secondMember.ref)
 
-	amount := 111
+	amount := "10"
 
-	_, err := signedRequest(firstMember, "Transfer", amount, secondMember.ref)
-	require.Contains(t, err.Error(), "not found")
+	_, err := signedRequest(firstMember, "wallet.transfer", map[string]interface{}{"amount": amount, "to": secondMember.ref})
+	require.NotNil(t, err)
```

Comment:
Its better to check error message here also, to be sure its about not existed member, not smth else